src/test/java/edu/hawaii/its/api/service/UpdateMemberServiceTest.java

```diff
@@ -433,4 +497,55 @@ public void optOutTest() {
         assertNotNull(updateMemberService.optOut(TEST_USERNAMES.get(0), groupPath, TEST_USERNAMES.get(1)));
     }
 
+
+/*
+    @Test
+    public void removeOwnerTest() {
+        String json = propertyLocator.find("ws.has.member.results.is.members.uhuuid");
+        WsHasMemberResults wsHasMemberResults = JsonUtil.asObject(json, WsHasMemberResults.class);
+        HasMembersResults hasMembersResults = new HasMembersResults(wsHasMemberResults);
+        assertNotNull(hasMembersResults);
+        doReturn(hasMembersResults).when(grouperApiService).hasMemberResults(GroupType.OWNERS.value(), TEST_USERNAMES.get(1));
+
+        json = propertyLocator.find("ws.get.subject.result.success");
+        WsGetSubjectsResults wsGetSubjectsResults = JsonUtil.asObject(json, WsGetSubjectsResults.class);
+        SubjectsResults subjectsResults = new SubjectsResults(wsGetSubjectsResults);
+        assertNotNull(subjectsResults);
+        doReturn(subjectsResults).when(grouperApiService).getSubjects(TEST_USERNAMES.get(0));
+
+        json = propertyLocator.find("ws.delete.member.results.success");
+        WsDeleteMemberResults wsDeleteMemberResults = JsonUtil.asObject(json, WsDeleteMemberResults.class);
+        RemoveMemberResult removeMemberResult = new RemoveMemberResult(wsDeleteMemberResults.getResults()[0], groupPath);
+        assertNotNull(removeMemberResult);
+        doReturn(removeMemberResult).when(grouperApiService)
+                .removeMember(TEST_USERNAMES.get(1), GroupType.OWNERS.value(), TEST_USERNAMES.get(0));
+
+        assertNotNull(updateMemberService.removeOwnership(TEST_USERNAMES.get(1), TEST_USERNAMES.get(0)));
+
+        json = propertyLocator.find("ws.get.subject.result.uid.failure");
+        wsGetSubjectsResults = JsonUtil.asObject(json, WsGetSubjectsResults.class);
+        subjectsResults = new SubjectsResults(wsGetSubjectsResults);
+        assertNotNull(subjectsResults);
+        doReturn(subjectsResults).when(grouperApiService).getSubjects("bogus-identifier");
+
+        assertThrows(UhMemberNotFoundException.class,
+                () -> updateMemberService.removeAdminMember(TEST_USERNAMES.get(1), "bogus-identifier"));
+    }
+
+
+ */
```

Comment:
I believe we created this test but we were not sure if it should be included in this PR. Let's uncomment this and include it.

Reply:
I will remove it since it is an old version I used for testing